src/shadertools/src/utils/webgl-info.js

```diff
@@ -0,0 +1,125 @@
+// Feature detection for WebGL
```

Comment:
So the intention to duplicate `hasFeature` functionality is that `shadertools` module lives below luma.gl in stack?

Reply:
Yes, there is some duplication, but we are only duplicating a small part of it. I think we should rename the function to reduce confusion. I will come back to that in another PR.